### PR TITLE
panel update via instrument

### PR DESCRIPTION
**Pull request by joelvbernier**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 np_include_dir = os.path.join(numpy.get_include(), 'numpy')
 
 install_reqs = [
-    'fabio@git+https://github.com/joelvbernier/fabio.git@master',
```
Comment (cjh1) [inline setup.py:11]: Are you changes now in a release, I didn't see one cut?

Comment (joelvbernier) [reply setup.py:11]: Hmmm...  maybe not.  Just asked the silx maintainer when we can expect one.

Comment (cjh1) [reply setup.py:11]: The rest of the changes look good, so if you want to remove this change, we can merge? Or wait for fabio ...

Comment (joelvbernier) [reply setup.py:11]: Yeah, they are busy with the restart at ESRF, so it might be a bit...  I'll change it back.

**Pull request merged**